.travis.yml

```diff
@@ -0,0 +1,25 @@
+env:
+  - NODE_VERSION=0.10
+  - NODE_VERSION=0.12
+
+os:
+  - linux
+  - osx
+
+install:
+  - git clone https://github.com/creationix/nvm.git /tmp/.nvm
+  - source /tmp/.nvm/nvm.sh
+  - nvm install $NODE_VERSION
+  - nvm use $NODE_VERSION
+  - if [ $TRAVIS_OS_NAME == "linux" ]; then
+      sudo apt-get install build-essential git libgnome-keyring-dev fakeroot;
+      export DISPLAY=:99.0;
+      sh -e /etc/init.d/xvfb start;
```

Comment:
If we decide not to run specs on Linux, I think these last two lines may be commented out.